app/model/BaseRule.scala

```diff
@@ -51,18 +51,20 @@ case class RegexRule(
 
   def toMatch(start: Int, end: Int, block: TextBlock): RuleMatch = {
     val matchedText = block.text.substring(start, end)
-
+    val (before, after) = Text.getSurroundingText(block.text, start, end)
```

Comment:
If we can find a wording that is clearer, I'd prefer it?  Leading / trailing?

Reply:
Changed to precedingText/subsequentText, thanks for pointing out and to @twrichards for the suggestion 👍